xml/soc-operations-maintenance.xml

```diff
@@ -1444,8 +1442,49 @@
     <para>
      Perform a rolling upgrade to &mariadb; following the instructions at <link
      xlink:href="https://mariadb.com/kb/en/library/upgrading-between-minor-versions-with-galera-cluster/">Upgrading
-     Between Minor Versions with Galera Cluster</link>. Each node must upgraded
-     individually so that the cluster is always operational.
+     Between Minor Versions with Galera Cluster</link>.
+    </para>
+    <para>
+     In general, the process involves the following steps:
```

Comment:
```suggestion
The process involves the following steps:
```

Reply:
Question about this - was there a request to include the generalised instructions even though we link to the proper set? Seems redundant to rewrite it.

Reply:
yes. Lots of comments on the bug and Rocket Chat suggestions too. The process apparently is confusing as it happens. Someone? suggested giving a high level view.

Reply:
Huh fair enough - makes sense then :)